docs/FOR VALIDATORS/Installation & Quick Start/imversed.md

```diff
@@ -0,0 +1,166 @@
+* `everything`: Prune all saved states other than the current state.
+* `nothing`: Save all states and delete nothing.
+* `default`: Save the last 100 states and the state of every 10,000th block.
+* `custom`: Specify pruning settings with the `pruning-keep-recent`, `pruning-keep-every`, and `pruning-interval` parameters.
+
+By default, every node is in `default` mode which is the recommended setting for most environments. If you would like to change your nodes pruning strategy then you must do so when the node is initialized. Passing a flag when starting `imversed` will always override settings in the `app.toml` file, if you would like to change your node to the `everything` mode then you can pass the `---pruning everything` flag when you call `imversed` start.
+
+> **IMPORTANT:** When you are pruning state you will not be able to query the heights that are not in your store.
+
+### Client configuration
+We can view the default client config setting by using `imversed config` command:
+
+```text
+imversed config
+{
+ "chain-id": "",
+ "keyring-backend": "os",
+ "output": "text",
+ "node": "tcp://localhost:26657",
+ "broadcast-mode": "sync"
+}
+```
+We can make changes to the default settings upon our choices, so it allows users to set the configuration beforehand all at once, so it would be ready with the same config afterward.
+
+For example, the chain identifier can be changed to `imversed_9000-4` from a blank name by using:
+
+```text
+imversed config "chain-id" imversed_9000-4
+imversed config
+{
+ "chain-id": "imversed_9000-4",
+ "keyring-backend": "os",
+ "output": "text",
+ "node": "tcp://localhost:26657",
+ "broadcast-mode": "sync"
+}
+```
+
+Other values can be changed in the same way.
+
+Alternatively, we can directly make the changes to the config values in one place at client.toml. It is under the path of `.imversed/config/client.toml` in the folder where we installed imversed:
+
+```text
+############################################################################
+### Client Configuration ###
+
+############################################################################
+
+# The network chain ID
+
+chain-id = "imversed_9000-4"
+
+# The keyring's backend, where the keys are stored (os|file|kwallet|pass|test|memory)
+
+keyring-backend = "os"
+
+# CLI output format (text|json)
+
+output = "number"
+
+# <host>:<port> to Tendermint RPC interface for this chain
+
+node = "tcp://localhost:26657"
+
+# Transaction broadcasting mode (sync|async|block)
+
+broadcast-mode = "sync"
+```
+
+After the necessary changes are made in the `client.toml`, then save. For example, if we directly change the chain-id from `imversed_9000-2` to `imversedtest_9000-1`, and output to number, it would change instantly as shown below.

```

Comment:
Change chain-id:
- `imversed_9000-2` to `imversed_5555555-1`
- `imversedtest_9000-1` to `imversedtest_5555555-6`

Reply:
Updated...